com.jianguoyun.Nutstore.json

```diff
@@ -0,0 +1,69 @@
+{
+	"app-id": "com.jianguoyun.Nutstore",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "nutstore",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--share=network",
+		"--filesystem=home"
+	],
+
+	"modules": [
+		{
+			"name": "nutstore",
+			"only-arches": ["x86_64"],
+			"buildsystem": "simple",
+			"build-commands": [
+				"install -D -t /app/bin -m 755 nutstore",
+				"install -D -t /app/bin -m 755 apply_extra",
+				"install -D -t /app/share/applications -m 644 com.jianguoyun.Nutstore.desktop",
+				"install -D -m 644 com.jianguoyun.Nutstore.64x64.png /app/share/icons/hicolor/64x64/apps/com.jianguoyun.Nutstore.png",
+				"install -D -m 644 com.jianguoyun.Nutstore.128x128.png /app/share/icons/hicolor/128x128/apps/com.jianguoyun.Nutstore.png",
+				"install -D -t /app/share/metainfo -m 644 com.jianguoyun.Nutstore.metainfo.xml"
+			],
+			"sources": [
+				{
+					"type": "extra-data",
```

Comment:
Please add x-checker-data so it's automatically updated, see https://github.com/flathub/flatpak-external-data-checker/

Reply:
Yeah, it's actually broken already:

```
Error: Wrong size for extra data https://www.jianguoyun.com/static/exe/installer/nutstore_linux_dist_x64.tar.gz
error: Failed to install com.jianguoyun.Nutstore: Wrong size for extra data https://www.jianguoyun.com/static/exe/installer/nutstore_linux_dist_x64.tar.gz
```